### ISSUE TITLE
Protocol for making suggestions back to USWDS

### ISSUE THREAD

**Issue by bruce-usab** [atbcb/usab-uswds#574]
Kathy has made some improvements (for accessibility) that should be passed along to the USWDS folks
- [ ] external link, see PR https://github.com/atbcb/usab-uswds/issues/248
- [x] skip to nav bar as well as skip to main content
- [ ] about menu and similar from template

**Comment by kengdoj** [comment]
- [ ] keyboard focus to cards (like on Board Members page)

**Comment by bruce-usab** [comment]
From default Jekyll build from template, `_includes/searchgov/form.html` line 17 `label for` has a typo:
`<label class="usa-sr-only" for="extended-search-field-small">Search small</label>`

Should just be:
`<label class="usa-sr-only" for="extended-search-field-small">Search small</label>`

**Comment by bruce-usab** [comment]
I have gotten positive email feedback from @afeijoo but my preference would be to submit our suggestions as pull requests.  Assuming the bug fixes slow down in a couple weeks, I would like help navigating to the right repo.

**Comment by afeijoo** [comment]
@bruce-usab You can submit issues and PRs to https://github.com/uswds/uswds.

**Comment by bruce-usab** [comment]
@drewnielson recently reminds that we also didn't find the visual indication of one's location in main navigation menus to be sufficiently accessible.  There is good visual highlighting, but no character-based indication, so nothing for someone using screen reading software to easily identify.  We adding a right-pointing arrow → but might have used an asterisk or bullet or greater-than.  The &rarr; is an html element, but not easily typed from a keyboard.

For a good example of our implementation, please see:  https://www.access-board.gov/about/
